pages/1.13/api/logs.yaml

```diff
@@ -3,15 +3,7 @@ swagger: '2.0'
 info:
   version: "0.0.1"
   title: DC/OS Log API v1
-  description: >
```

Comment:
Are we expecting this description to be removed here?

Reply:
I can't answer that. Can you ask Engineering?